src/main/java/uk/gov/hmcts/sptribs/services/cdam/CaseDocumentApiService.java

```diff
@@ -21,9 +24,12 @@ public class CaseDocumentApiService {
     @Autowired
     DssCaseDocumentClient dssCaseDocumentClient;
 
+
     public DocumentInfo uploadDocument(String authorizationToken, MultipartFile file,
                                        AppsConfig.AppsDetails appsDetails) {
 
+        log.info("uploadDocument start");
```

Comment:
Can be changed to debug or removed to save cost.

Reply:
We need the logs at least temporarily to so that platops can help us work out what is happening with the document upload issue.

Reply:
Yep, already have done this